telescope/query.py

```diff
@@ -257,9 +270,19 @@ def _add_server_ips_conditional(self, server_ips, is_web100):
     else:
       local_ip_fieldname = 'connection_spec.server_ip'
 
-    self._conditional_dict['server_ip'] = []
+    self._conditional_dict['server_ips'] = []
     for server_ip in unique_server_ips:
       new_statement = '{local_ip_fieldname} = \'{server_ip}\''.format(
           local_ip_fieldname=local_ip_fieldname, server_ip=server_ip)
-      self._conditional_dict['server_ip'].append(new_statement)
-
+      self._conditional_dict['server_ips'].append(new_statement)
+
+  def _add_client_country_conditional(self, client_country):
+    self._conditional_dict['client_country'] = ("{fieldname} = " +
+        "UPPER('{client_country}')").format(fieldname =
```

Comment:
This is the only case where it needs to be uppercase, whereas in other cases (e.g. filenames) it should be lower or it doesn't matter, so I am going to normalize them to lower in the Selector parser and then pass the value to the query builder with .upper() beforehand.